tools/astarte_e2e/lib/astarte_e2e/service_notifier.ex

```diff
@@ -101,8 +105,13 @@ defmodule AstarteE2E.ServiceNotifier do
   end
 
   def service_up({:call, from}, {:notify_service_down, reason}, data) do
-    mail = Email.service_down_email(reason)
-    deliver(mail, reason)
+    reason
+    |> Email.service_down_email()
+    |> deliver()
+
+    Logger.warn("Service down. The user has been notified. Reason: #{reason}",
+      tag: "service_down_notified"
+    )
```

Comment:
I suppose this should be `service_up_email` and "Service is back up" etc etc

Also, this should be `info`, not `warn`.